test/factory/user_factory.js

```diff
@@ -0,0 +1,59 @@
+const { factory } = require('factory-girl');
+const Chance = require('chance');
+const { factoryByModel } = require('./factory_by_models');
+const { hash } = require('.././../app/helpers/auth');
+const { badRequest } = require('../../app/errors');
+const { responseError } = require('../helpers/errors');
+const { removeProperty } = require('../helpers/body');
+
+const chance = new Chance();
+factoryByModel('User');
+
+factory.extend('User', 'UserWithHash', {
+  password: hash('12345678wwwffC'),
+  email: factory.sequence('User.email', n => `user.test${n}@wolox.com.co`)
+});
+
+const userFake = () => ({
+  name: chance.first(),
+  surname: chance.last(),
+  email: chance.email({ domain: 'accenture.com' }),
+  password: 'XYZ1234567789'
+});
+
+const newUser = userFake();
+exports.UserData = {
+  newUser,
+  responseWhenUserIsCreated: {
+    surname: newUser.surname,
+    name: newUser.name,
+    email: newUser.email
+  },
+  userPasswordNumeric: {
+    ...userFake(),
+    password: '123456789000'
+  },
+  userPasswordMinLength: {
+    ...userFake(),
+    password: 'ww123'
+  },
+  userPasswordAlphabetic: {
+    ...userFake(),
+    password: 'abcdefghijkl'
+  },
+  userWithoutEmail: removeProperty('email')(userFake()),
+  userWithoutName: removeProperty('name')(userFake()),
+  userWithoutSurname: removeProperty('surname')(userFake()),
+  userWithoutPassword: removeProperty('password')(userFake()),
+  userWithoutParameters: {}
+};
+
+exports.UserErrors = {
```

Comment:
Why is this in PascalCase? We use PascalCase for clases or models (that works kind of like clases)

Reply:
Sorry, my mistake, I replaced it with lowerCamelCase.